docker/jmeter-master/Dockerfile

```diff
@@ -9,10 +9,6 @@ ENV TESTS_DIR /tests
 ENV SLEEP 5
 ENV USE_WORKERS false
 
-ENV AWS_DEFAULT_REGION ""
```

Comment:
if we want to keep backward compatibility, shall we keep those vars?

Reply:
removing this won't break BC, I think it will better to remove, so new users don't get confused with those ENVs.